models/model1_part2.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<p>Keep the imports together.</p><p>pandas has already been imported earlier.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>